outlook/src/taskpane/components/Log/Logger.tsx

```diff
@@ -27,21 +34,141 @@ class Logger extends React.Component<LoggerProps, LoggerState> {
         };
     }
 
-    private logRequest = (event) => {
+    private fetchAttachmentContent(attachment, index): Promise<any> {
+        return new Promise<any>((resolve) => {
+            if (attachment.size > elementSizeThreshold * 1024 * 1024) {
+                if (attachment.isInline && attachment.contentType.indexOf("image") >= 0) {
+                    resolve({
+                        name: attachment.name, inline: true,
+                        oversize: true, index
+                    });
+                } else {
+                    resolve({
+                        name: attachment.name, inline: false,
+                        oversize: true, index
+                    });
+                }
```

Comment:
Since those 2 are very similar, I suggest:

```js
resolve({
    name: attachment.name,
    inline: attachment.isInline && attachment.contentType.indexOf("image") >= 0,
    oversize: true,
    index: index  // <-- I see you use this syntax here below so let's be consistent
});
```

When building dicts, I believe having a single key per line makes it easier to read.

Side note: you can apply the same remark for the code here below.

Reply:
Yes indeed, it has now been corrected